jetty-core/jetty-quic/jetty-quic-quiche/jetty-quic-quiche-common/src/main/java/org/eclipse/jetty/quic/quiche/QuicheBinding.java

```diff
@@ -20,7 +20,7 @@
 
 public interface QuicheBinding
 {
-    boolean isUsable();
+    Throwable checkUsable();
```

Comment:
Rename to `Throwable initialize()`, since it calls `initialize()` on the native implementation.

Reply:
Done.